client-spark/common/src/main/java/org/apache/spark/shuffle/RssClientConfig.java

```diff
@@ -79,4 +79,13 @@
   // has huge amount blockIds, eg, 10B.
   public static String RSS_CLIENT_BITMAP_SPLIT_NUM = "spark.rss.client.bitmap.splitNum";
   public static int RSS_CLIENT_BITMAP_SPLIT_NUM_DEFAULT_VALUE = 1;
+  public static String RSS_ACCESS_ID_KEY = "spark.rss.accessIdKey";
+  public static String RSS_ACCESS_ID_KEY_DEFAULT_VALUE = "";
+  public static String RSS_ACCESS_ID_DEFAULT_VALUE = "";
```

Comment:
What's the difference between `RSS_ACCESS_ID_KEY` and `RSS_ACCESS_ID_DEFAULT_VALUE`?

Reply:
This is kind of specific, I will remote it.